static/mergify-count-active-users.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# This scripts counts the number of active users for repositories over the last 30 days.
+# See https://docs.mergify.com/billing/ for context.
+#
+# Keep in mind that this is not 100% accurate, but gives a ballpark estimate.
+#
+# You'll need Python 3 and requests for this to work.
+# You can install requests by running:
+# $ pip install requests
+# then run this script with
+# $ python3 mergify-count-active-users <url of your repo>
+
+import collections
+import requests
+import pprint
+import os
+from datetime import datetime, timedelta
+from urllib.parse import urlparse
+
+LOOKBACK_WINDOW = timedelta(days=30)
+
+HEADERS = {
+    "Authorization": f"token {os.environ['GITHUB_TOKEN']}",
```

Comment:
Won't this still crash, even with the check in the first if, if the user forgot the env variable ?

Reply:
good point